security/acme-client/src/opnsense/mvc/app/models/OPNsense/AcmeClient/AcmeClient.xml

```diff
@@ -612,6 +613,16 @@
                 <dns_linode_key type="TextField">
                   <Required>N</Required>
                 </dns_linode_key>
+                <dns_loopia_api type="TextField">
+                  <Required>N</Required>
+                  <default>https://api.loopia.se/RPCSERV</default>
```

Comment:
with a default it would make sense to make it required.

Reply:
We cannot have required values there. They would still be "required" even if one chooses a different DNS-01 provider (not "loopia").